### PR TITLE
feat: FAQ - Add Help Center

### PR DESCRIPTION
**Pull request by fhildeb**
Removes old FAQ Help page and sets up the new help center.

### CODE REVIEW

File: docs/faq/help-center.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (Hugoo) [inline docs/faq/help-center.md:1]: I'm not sure it is necessary to change the file name (which impacts the URL).

If you do change a page URL, I highly recommend to add a redirect:

https://github.com/lukso-network/docs/blob/main/docusaurus.config.js#L19

This is for SEO purposes and also to make sure all exising links to this page don't go to a 404.

But i think we can keep "help.md" (so `/help` URL)

File: docs/faq/help-center.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+sidebar_label: 'Help Center'
+sidebar_position: 5
+---
+
+# Help Center
+
+### How to stay up to date about LUKSO?
```
Comment (Hugoo) [inline docs/faq/help-center.md:8]: why go from h1 to h3 directly without h2?

File: docs/faq/help-center.md (hunk 1 above)
Comment (CJ42) [reply docs/faq/help-center.md:1]: Agree on this

File: docs/faq/help-center.md (hunk 2 above)
Comment (CJ42) [reply docs/faq/help-center.md:8]: Yes let's keep h2 so ##, so that we also have a good ToC on the right sidebar

File: docs/faq/help-center.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+You can follow us on [Twitter](https://twitter.com/lukso_io), read our [Medium](https://medium.com/lukso) articles, and follow the announcements on our [Discord](https://discord.gg/lukso) server.
```
Comment (CJ42) [inline docs/faq/help-center.md:10]: Suggesting to not use "our" but keep it general.

```suggestion
You can follow LUKSO on [Twitter](https://twitter.com/lukso_io), read the latest article on [Medium](https://medium.com/lukso) articles, and follow the announcements on the LUKSO [Discord](https://discord.gg/lukso) server.
```

File: docs/faq/help-center.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+### Where can I read LUKSOs Whitepaper?
```
Comment (CJ42) [inline docs/faq/help-center.md:12]:
```suggestion
### Where can I read the LUKSO Whitepaper?
```

File: docs/faq/help-center.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Here is the link to read and download the [Official LUKSO Whitepaper](https://uploads-ssl.webflow.com/629f44560745074760731ba4/62b200bfe0af12186845519a_LUKSO_Whitepaper_V1-1.pdf).
+
+### My question is not here. Where can I ask it?
+
+If you can't find an answer to your question, you can head to our [Discord](https://discord.gg/lukso) server. We have development-related channels where you can ask technical questions. We will add the most asked question to this list.
```
Comment (CJ42) [inline docs/faq/help-center.md:18]: Might be worth mentioning the channels here, related to specific topics?

File: docs/faq/help-center.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+### Where can I report regular issues for repositories and tools?
+
+We are preparing workflows to handle issues, investigations, and bug bounties. For now, please create an issue on our global [GitHub Issue Tracker](https://github.com/lukso-network/issue-tracker).
```
[the remark this answers is not in the document]
Comment (CJ42) [reply docs/faq/help-center.md:22]: Yes we should encourage people to post issues in the actual repos.

For the browser extension, I would suggest the issue tracker link mentioned here.

File: docs/faq/help-center.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+### Where can security issues be reported to?
+
+Suppose it's a vulnerability or something that should not be shared with the public. Please reach out to official team members within our developer channel on [Discord](https://discord.gg/lukso).
```
Comment (CJ42) [inline docs/faq/help-center.md:26]: We should suggest that they reach out to the LUKSO team on these server, send us a friend request and let us know by DM.

File: docs/faq/help-center.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+### Does LUKSO have grants or bug bounties?
+
+We're happy to compensate if there are severe findings and vulnerabilities discovered. The compensation will depend on the scope of the error or bug. We do not share precise bounty amounts beforehand because of the wide range of possible discoveries.
```
Comment (CJ42) [inline docs/faq/help-center.md:30]: We need to check this info with the business team before hand.
Please check with Jonathan.

**Review by CJ42 — COMMENTED** [review]
Added review comments

File: docs/faq/help-center.md (hunk 8 above)
Comment (fhildeb) [reply docs/faq/help-center.md:30]: I have checked this with Fabian directly

**Comment by fhildeb** [comment]
Thank you, guys. 👍🏻 I resolved all comments and pushed the changes.

**Review by CJ42 — APPROVED** [review]

**Comment by Hugoo** [comment]
Nice thks for handling the changes, let's merge 🚀

**Pull request merged**